resources/schema.py

```diff
@@ -233,28 +233,38 @@ class Meta:
         )
 
 
-class HarborNode(graphql_geojson.GeoJSONType):
+class HarborFilter(django_filters.FilterSet):
     class Meta:
         model = Harbor
-        filter_fields = [
+        fields = (
             "piers__mooring",
             "piers__electricity",
             "piers__water",
             "piers__waste_collection",
             "piers__gate",
             "piers__lighting",
             "piers__suitable_boat_types",
-            "maximum_width",
-            "maximum_length",
-        ]
+        )
+
+    max_width = django_filters.NumberFilter()
+    max_length = django_filters.NumberFilter()
+
+
+class HarborNode(graphql_geojson.GeoJSONType):
+    class Meta:
+        model = Harbor
         geojson_field = "location"
         interfaces = (relay.Node,)
+        filterset_class = HarborFilter
 
     name = graphene.String()
     street_address = graphene.String()
     municipality = graphene.String()
     image_file = graphene.String()
     maps = graphene.List(HarborMapType, required=True)
+    max_width = graphene.Float()
+    max_length = graphene.Float()
```

Comment:
we could mark these as `required=True`

Reply:
OTOH, hm, what if a harbor has no berths. like if we just created it. will the `Max` annotations return `null`s? if so, maybe it's not a bad idea to have these as just `graphene.Float()` 🤔

Reply:
I think it's better to keep it as `required=False` unless we want to return `0` instead of `null`. 🤔

Reply:
yeah, let's keep it as is 👍 I think it makes more sense if we return `null` when there are no berths in a harbor